besu/src/main/java/org/hyperledger/besu/controller/CliqueBesuControllerBuilder.java

```diff
@@ -104,6 +108,22 @@ protected MiningCoordinator createMiningCoordinator(
       miningCoordinator.enable();
     }
 
+    if (MergeConfiguration.isMergeEnabled()) {
+      final BackwardsSyncContext backwardsSyncContext =
+          new BackwardsSyncContext(
+              protocolContext, protocolSchedule, metricsSystem, ethProtocolManager.ethContext());
+      final MergeCoordinator mergeCoordinator =
+          new MergeCoordinator(
+              protocolContext,
+              protocolSchedule,
+              transactionPool.getPendingTransactions(),
+              miningParameters,
+              backwardsSyncContext);
+      final TransitionCoordinator mergeReadyClique =
+          new TransitionCoordinator(miningCoordinator, mergeCoordinator);
+      return mergeReadyClique;
```

Comment:
Unless I am missing something, I don't think we should be doing this.  the CliqueBesuControllerBuilder should just return a clique coordinator.  If the merge is enabled, we are automatically composing the controller builders in [the TransitionControllerBuilder ](https://github.com/hyperledger/besu/blob/merge/besu/src/main/java/org/hyperledger/besu/controller/TransitionBesuControllerBuilder.java#L84-L99)

was this necessary because of the execution endpoint addition?  I think the net result is that we are going to have a 'double wrapped' clique besu controller when merge is enabled